doc/man/man8/daos.8

```diff
@@ -124,6 +126,11 @@ The \fBRESOURCE\fRs, respective \fBCOMMAND\fRs and \fBOPTION\fRs supported by \f
 .br
 	  \fB--cont=\fRUUID      (optional) container UUID (or generated)
 .TP
+.I container \fBOPTION\fRs (copy a container):
+	  \fB--src=\fRSTR      <pool uuid>/<cont uuid> | <UNS path>
+.br
+	  \fB--dst=\fRSTR      <pool uuid>/<cont uuid> | <UNS path>
```

Comment:
is this always required?
let's say one wants to copy a container but wants us to generate the uuid for it.. same like when we call daos cont create. the uuid for the container is optional.